activerecord/CHANGELOG.md

```diff
@@ -34,7 +60,7 @@
 
     *Luis Vasconcellos*, *Eileen M. Uchitelle*
 
-*   Fix `eager_loading?` when ordering with `Hash` syntax.
+*   Fix `eager_loading?` when ordering with `Hash` syntax
```

Comment:
@kaspth I think this change was unintended.
I've made a PR to add the dot back (and add some others).
https://github.com/rails/rails/pull/42967

Reply:
@p8 Indeed, thanks!